lib/Common/Memory/Recycler.h

```diff
@@ -1921,6 +1921,15 @@ class Recycler
     } objectBeforeCollectCallbackState;
 
     bool ProcessObjectBeforeCollectCallbacks(bool atShutdown = false);
+
+#if DBG
+private:
+    static Recycler* recyclerList;
+    Recycler* next;
```

Comment:
just in case, there's multiple thread context we'll have multiple recycler, aka, node case?

---
In reply to: [92457813](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92457813) [](ancestors = 92457813)